styles.css

```diff
@@ -1,85 +1,262 @@
-@import url('https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Poppins:wght@200;300;400;500;600;800&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Inter:wght@200;300;400;500;600;700&family=Poppins:wght@200;300;600;800&family=Roboto:wght@700&display=swap');
 
 body { margin: 0%; }
 
 .container {
   display: flex;
   flex-direction: column;
-  border-radius: 2%;
   width: 375px;
   height: 100vh;
-  background-color: black;
+  background-color: #1c1a19;
+  background-image: url("illustration-header-mobile.svg");
+  background-repeat: no-repeat;
+  background-position: bottom right;
 }
-.icon { height: 70px; }
 
 .navigation {
   display: flex;
   justify-content: space-around;
   background-color: rgb(34, 33, 33);
 }
+.list { padding-left: 0%; }
```

Comment:
- [ ] Kindly make sure you stick to the original design.
`hint: you can use justify-content: space-between` on your hear 
As you can see your design :point_down: 
![Screenshot from 2022-09-25 00-27-35](https://user-images.githubusercontent.com/97100091/192120915-62ac5044-4a6c-4e28-b55f-5911aebda395.png)
 versus the Figma template
![Screenshot from 2022-09-25 00-30-33](https://user-images.githubusercontent.com/97100091/192120943-a78eb9e8-6360-4043-96e7-e34f9576c289.png)